app.py

```diff
@@ -1,11 +1,15 @@
 from flask import Flask
 from flask import render_template
 from flask import request
+import datasets
+import transformers
 
 app = Flask(__name__)
 
 def summarize(s): # make this an actual summarize function
-    return s[::2]
+    summarizer = transformers.pipeline("summarization", model = "t5-small")
```

Comment:
Going to merge this PR, but if you do this we need to load the model each time. Maybe explore ways to load it once